### ISSUE TITLE
Saving new role fails with not null contraint violation

### ISSUE THREAD

**Issue by ejortegau** [Zizaco/entrust#58]
When creating a new role as describe in Usage > concepts section of the docs:

$admin = new Role;
$admin->name = "Administrador";
$admin->save();

And running it as part of my seeder, I get this error:

[Exception]  
  SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "permissions" violates  
  not-null constraint (SQL: insert into "roles" ("name", "updated_at", "created_at") values (  
  ?, ?, ?) returning "id") (Bindings: array (  
    0 => 'Administrador',  
    1 => '2013-10-28 04:58:22',  
    2 => '2013-10-28 04:58:22',  
  ))

So basically it does not work as specified in the docs.
